Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeNode.h

```diff
@@ -48,6 +48,27 @@ OBJC_CLASS UIView;
 
 namespace WebKit {
 
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+class RemoteAcceleratedEffectStack : public ThreadSafeRefCounted<RemoteAcceleratedEffectStack>, public WebCore::AcceleratedEffectStackBase
```

Comment:
I think I'd move this class into its own source file if you agree with the suggestion made above about moving more methods to this specific subclass.

Reply:
Oh definitely, I was just being lazy while prototyping :)